runtime/onert/core/src/ir/train/TrainableGraph.cc

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright (c) 2023 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "ir/train/TrainableGraph.h"
+
+namespace onert
+{
+namespace ir
+{
+namespace train
+{
+
+TrainableGraph::TrainableGraph(const Graph &graph) : _graph{graph}, _operations{} {}
```

Comment:
I think it marks that I didn't miss out on the initialization.

Reply:
May I understand it is a mark for you? It is automatically done by compiler as I understand.

Reply:
I know it is automatically done by compiler. 
I mean to mark it so that anyone who didn't implemented this code recognized that `_operations` had been initialized.